gitlab-ls.py

```diff
@@ -227,6 +245,8 @@ def get_merge_request_dict(project: Project, updated_after: str = None) -> Dict[
 @server.feature(types.INITIALIZE)
 async def fetch_database(ls: GitlabLanguageServer, params: types.InitializeParams):
     init_options = params.initialization_options
+    if init_options is None:
+        return
```

Comment:
Issue an error/warning before exiting would be better. You can use `ls.show_message()`

Reply:
done